package.json

```diff
@@ -5,11 +5,17 @@
   "version": "0.1.0",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.21",
```

Comment:
Any specific reason to use this over something like `react-icons`? (I'm partial to it, as I've used it in the past, just curious)

Reply:
Didn't know that package existed, but it does seem a better alternative. Will make the change as soon as possible.

Reply:
:shipit: